behaviour/graql/reasoner/resolution/value-predicate.feature

```diff
@@ -79,8 +78,7 @@ Feature: Value Predicate Resolution
         $a > 0;
       },
       then {
-        $x has is-old $t;
-        $t true;
+        $x has is-old true;
```

Comment:
was this the format of the test originally?

Reply:
This was one of the tests that we had to modify in order to make it pass, and was one of the incentives for this PR

Reply:
good